website/content/en/docs/upgrading/v1beta1-migration.md

```diff
@@ -199,10 +199,11 @@ If you are using some IaC for managing your policy documents attached to the con
 
 Karpenter v1beta1 introduces changes to some common labels, annotations, and status conditions that are present in the project. The tables below lists the v1alpha5 values and their v1beta1 equivalent.
 
-| Karpenter Labels                |                             |
-|---------------------------------|-----------------------------|
-| **v1alpha5**                    | **v1beta1**                 |
-| karpenter.sh/provisioner-name   | karpenter.sh/nodepool       |
+| Karpenter Labels                |                       |
+|---------------------------------|-----------------------|
+| **v1alpha5**                    | **v1beta1**           |
+| karpenter.sh/provisioner-name   | karpenter.sh/nodepool |
+| karpenter.k8s.aws/instance-pods | **Dropped**           |
```

Comment:
Hi @jonathan-innis. You'll get an error message that `karpenter.k8s.aws/instance-pods` is not well-known for `v1alpha5`, as well. It makes sense to write a sentence somewhere, that `karpenter.k8s.aws/instance-pods` is dropped since 0.32 in general.